src/js/components/DropButton/stories/typescript/Calendar.tsx

```diff
@@ -1,43 +0,0 @@
-import React, { useState } from 'react';
-import { storiesOf } from '@storybook/react';
-import { FormDown } from 'grommet-icons';
-import isChromatic from 'chromatic/isChromatic';
-
-import { Grommet, Box, Calendar, DropButton, Text } from 'grommet';
-import { grommet } from 'grommet/themes';
-
-const CalendarDropButton = () => {
```

Comment:
Let's keep the js file instead.

Reply:
Removed `Calendar.js` instead of `Calendar.tsx`